package.json

```diff
@@ -13,14 +13,13 @@
   "private": true,
   "dependencies": {
     "babel-core": "latest",
-    "babel-polyfill": "latest",
     "babel-preset-es2015": "latest",
     "babel-preset-es2016": "latest",
     "babel-preset-es2017": "latest",
     "babel-preset-stage-0": "latest",
     "chalk": "^1.1.3",
     "cheerio": "^0.20.0",
-    "core-js": "latest",
+    "core-js-bundle": "^3.0.0",
```

Comment:
if the package name is changing, shouldn’t that be communicated somehow?

Reply:
It's the package only with bundled version. Communicated - what do you mead? It's renamed in cases of usage.

Reply:
why do we need the bundled version here?

Reply:
For generation of pages with test cases? See the source code of `build.js`.